packages/keyutils/keyutils.spec

```
@@ -0,0 +1,78 @@
+Name: %{_cross_os}keyutils
+Version: 1.6.1
+Release: 1%{?dist}
+Summary: Linux key management utilities
+License: GPL-2.0-or-later AND GPL-2.1-or-later
+Url: http://people.redhat.com/~dhowells/keyutils/
+Source0: http://people.redhat.com/~dhowells/keyutils/keyutils-%{version}.tar.bz2
+Source1: keyutils-tmpfiles.conf
+Source2: request-key.conf
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Development package for building Linux key management utilities
+
+%description devel
+%{summary}.
+
+%prep
+%setup -n keyutils-%{version} -q
+
+%global keyutilsmake \
+%set_cross_build_flags \\\
+export CC=%{_cross_target}-gcc ; \
+%make_build \\\
+  NO_ARLIB=1 \\\
+  ETCDIR=%{_cross_sysconfdir} \\\
+  LIBDIR=%{_cross_libdir} \\\
```

nit: it's very unlikely that this will ever matter, but setting `PREFIX` will put the right paths in `libkeyutils.pc` (which we don't and probably won't use):
```suggestion
  PREFIX=%{_cross_prefix} \\\
  LIBDIR=%{_cross_libdir} \\\
```

---

I added the PREFIX here and compared `libkeyutils.pc` before and after the change, but it seems like the paths are already correct?

Before:

```
libdir=/x86_64-bottlerocket-linux-gnu/sys-root/usr/lib
includedir=/x86_64-bottlerocket-linux-gnu/sys-root/usr/include

Name: libkeyutils
Description: keyutils library
Version: 1.6.1
Cflags: -I${includedir}
Libs: -L${libdir} -lkeyutils
```

After:

```
libdir=/x86_64-bottlerocket-linux-gnu/sys-root/usr/lib
includedir=/x86_64-bottlerocket-linux-gnu/sys-root/usr/include

Name: libkeyutils
Description: keyutils library
Version: 1.6.1
Cflags: -I${includedir}
Libs: -L${libdir} -lkeyutils
```